cmd/kubectl-gadget/deploy.go

```diff
@@ -231,6 +233,14 @@ func init() {
 	deployCmd.PersistentFlags().StringVarP(
 		&publicKey,
 		"public-key", "", resources.InspektorGadgetPublicKey, "Public key used to verify the container image")
+	deployCmd.PersistentFlags().BoolVarP(
+		&verifyGadgets,
+		"verify-gadgets", "",
+		true,
+		"verify gadgets using the provided public key")
+	deployCmd.PersistentFlags().StringVarP(
+		&gadgetsPublicKey,
+		"gadgets-public-key", "", resources.InspektorGadgetPublicKey, "Public key used to verify the gadgets")
 	rootCmd.AddCommand(deployCmd)
```

Comment:
What's the plan to support multiple keys? Will this parameter accept a list? Will it be possible to enable / disable verification per registry/prefix?